docs/en/sql-reference/window-functions/index.md

```diff
@@ -80,8 +80,8 @@ These functions can be used only as a window function.
 - `nth_value(x, offset)` - Return the first non-NULL value evaluated against the nth row (offset) in its ordered frame.
 - `rank()` - Rank the current row within its partition with gaps.
 - `dense_rank()` - Rank the current row within its partition without gaps.
-- `lagInFrame(x)` - Return a value evaluated at the row that is at a specified physical offset row before the current row within the ordered frame.
-- `leadInFrame(x)` - Return a value evaluated at the row that is offset rows after the current row within the ordered frame.
+- `lagInFrame(x, offset)` - Return a value evaluated at the row that is at a specified physical offset row before the current row within the ordered frame.
+- `leadInFrame(x, offset)` - Return a value evaluated at the row that is offset rows after the current row within the ordered frame.
```

Comment:
Actually, `lag`/`leadInFrame` can take 1-3 arguments

Reply:
Two questions then:
1. Is there a specific format used to indicate optional arguments?
2. What is the optional third argument? The description only mentioned an offset.

Reply:
The third argument is a default value, used when the offset is outside the frame.